docs/docs/resources/postgresql.md

```diff
@@ -291,3 +291,103 @@ The output is similar to the following:
   "PGUSER": "pg-service-user"
 }
 ```
+
+## Creating a PostgreSQL read-only replica
+
+Read-only replicas can be used to reduce the load on the primary service by making read-only queries against the replica service. 
+
+To create a read-only replica for a PostgreSQL service, you create a second PostgreSQL service and use [serviceIntegrations](https://aiven.github.io/aiven-operator/api-reference/postgresql.html#spec.serviceIntegrations) to replicate data from your primary service.
+
+The example that follows creates a primary service and a read-only replica:
+
+1\. Create a new file named pg-read-replica.yaml with the following:
+
+```yaml
+apiVersion: aiven.io/v1alpha1
+kind: PostgreSQL
+metadata:
+  name: primary-pg-service
+spec:
+  # gets the authentication token from the `aiven-token` Secret
+  authSecretRef:
+    name: aiven-token
+    key: token
+
+  # add your project's name here
+  project: <your-project-name>
+
+  # add the cloud provider and plan of your choice
+  # you can see all of the options at https://aiven.io/pricing
+  cloudName: google-europe-west1
+  plan: startup-4
+
+  # general Aiven configuration
+  maintenanceWindowDow: friday
+  maintenanceWindowTime: 23:00:00
+  userConfig:
+    pg_version: '14'
+
+---
+
+apiVersion: aiven.io/v1alpha1
+kind: PostgreSQL
+metadata:
+  name: read-replica-pg
+spec:
+  # gets the authentication token from the `aiven-token` Secret
+  authSecretRef:
+    name: aiven-token
+    key: token
+
+  # add your project's name here
+  project: <your-project-name>
+
+  # add the cloud provider and plan of your choice
+  # you can see all of the options at https://aiven.io/pricing
+  cloudName: google-europe-west1
+  plan: startup-4
+
+  # general Aiven configuration
+  maintenanceWindowDow: friday
```

Comment:
```suggestion
  maintenanceWindowDow: saturday
```

Not sure, but probably this would make sense: to not have both instances go for maintenance at the same time.